packages/@aws-cdk/aws-stepfunctions-tasks/README.md

```diff
@@ -19,48 +19,55 @@ This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aw
 
 ## Table Of Contents
 
-- [Task](#task)
-- [Paths](#paths)
-  - [InputPath](#inputpath)
-  - [OutputPath](#outputpath)
-  - [ResultPath](#resultpath)
-- [Parameters](#task-parameters-from-the-state-json)
-- [Evaluate Expression](#evaluate-expression)
-- [Batch](#batch)
-  - [SubmitJob](#submitjob)
-- [CodeBuild](#codebuild)
-  - [StartBuild](#startbuild)
-- [DynamoDB](#dynamodb)
-  - [GetItem](#getitem)
-  - [PutItem](#putitem)
-  - [DeleteItem](#deleteitem)
-  - [UpdateItem](#updateitem)
-- [ECS](#ecs)
-  - [RunTask](#runtask)
-    - [EC2](#ec2)
-    - [Fargate](#fargate)
-- [EMR](#emr)
-  - [Create Cluster](#create-cluster)
-  - [Termination Protection](#termination-protection)
-  - [Terminate Cluster](#terminate-cluster)
-  - [Add Step](#add-step)
-  - [Cancel Step](#cancel-step)
-  - [Modify Instance Fleet](#modify-instance-fleet)
-  - [Modify Instance Group](#modify-instance-group)
-- [Glue](#glue)
-- [Lambda](#lambda)
-- [SageMaker](#sagemaker)
-  - [Create Training Job](#create-training-job)
-  - [Create Transform Job](#create-transform-job)
-  - [Create Endpoint](#create-endpoint)
-  - [Create Endpoint Config](#create-endpoint-config)
-  - [Create Model](#create-model)
-  - [Update Endpoint](#update-endpoint)
-- [SNS](#sns)
-- [Step Functions](#step-functions)
-  - [Start Execution](#start-execution)
-  - [Invoke Activity Worker](#invoke-activity)
-- [SQS](#sqs)
+- [Tasks for AWS Step Functions](#tasks-for-aws-step-functions)
+  - [Table Of Contents](#table-of-contents)
```

Comment:
what's the rationale for adding these? it's somewhat implied by the headers no?
the [CDK docs page](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-stepfunctions-tasks-readme.html#table-of-contents) renders based on headers so technically this TOC is more of a convenience.